addon/components/g-map-polyline.js

```diff
@@ -75,20 +77,26 @@ const GMapPolylineComponent = Ember.Component.extend({
   setPath() {
     const polyline = this.get('polyline');
     const coordinates = this.get('coordinates');
-    const options = compact(this.getProperties(allowedPolylineOptions));
-    let coordArray = Ember.A(this.get('coordinates').mapBy('coordinate')).compact();
-    if (coordArray.length > 1 && isPresent(polyline) && isPresent(coordinates)) {
```

Comment:
looks like it works better without that check `coordArray.length > 1` - take a look at the dummy app in tests, I've added a button to toggle coord components.